drivers/hmis_data_quality_tool/app/views/hmis_data_quality_tool/warehouse_reports/reports/pdf_footer.haml

```diff
@@ -0,0 +1,62 @@
+= inline_stylesheet_link_tag 'dashboard_pdf'
+= inline_javascript_include_tag 'pdf'
+= inline_javascript_include_tag 'application'
+%link{href: '//fonts.googleapis.com/css?family=Open+Sans:300,400,400italic,600,700|Open+Sans+Condensed:700', rel: 'stylesheet', type: 'text/css'}
+.footer
+  .d-flex
+    .ml-auto
+      page
+      %strong
+        %span.pageNumber
+        of
+        %span.totalPages
+= svg 'symbol-defs'
+:css
+  .d-flex {
+    align-items: flex-start;
+    justify-content: flex-start;
+    flex-shrink: 1;
+  }
+  .d-flex .ml-2 {
+    align-items: flex-start;
+    justify-content: flex-start;
+    flex-shrink: 1;
+  }
+  .footer {
+    font-size: 10px;
+    position: absolute;
+    top: 9.6in;
+    width: 7in;
+    left: 0.25in;
+  }
+  .footer .icon-svg--sm svg, .footer .icon-svg svg {
+    height: 1rem;
+    width: 1rem;
+  }
+  p {
+    font-size: 16px !important;
+  }
+  .icon-info {
+    display: none;
+  }
+  a {
+    color: #333 !important;
+    text-decoration: none !important;
+  }
+  .perf-overview-breakdown {
```

Comment:
oh, you even have this here.  I _think_ most of this should be coming from the layout file and is already there.

Reply:
but maybe I'm missing something, this is just for the footer...